src/components/Avatar.tsx

```diff
@@ -49,6 +49,9 @@ type AvatarProps = {
 
     /** Owner of the avatar. If user, displayName. If workspace, policy name */
     name?: string;
+
+    /** ID of the Icon */
+    iconID?: number | string;
```

Comment:
```suggestion
    /** ID of the Icon */
    policyID?: number | string;
```

Reply:
Or are we passing the accountID otherwise?

Reply:
@grgia i updated, please check again